tests/component/test_task.py

```diff
@@ -194,3 +203,10 @@ def test___default_arguments___perform_thrmcpl_lead_offset_nulling_cal___no_erro
     ai_thermocouple_task: nidaqmx.Task,
 ) -> None:
     ai_thermocouple_task.perform_thrmcpl_lead_offset_nulling_cal()
+
+
+def test___is_task_done___task_is_done___returns_true(ai_task: nidaqmx.Task):
+    assert ai_task.is_task_done()
+    ai_task.start()
+    ai_task.wait_until_done()
+    assert ai_task.is_task_done()
```

Comment:
Huh, I'm sorta surprised this works. But maybe simulated devices always return True? Oh, maybe its actually On Demand that does that. TIL! Maybe rename `ai_task` in this file to `ai_on_demand_task` to better document why this works.